sync-core/src/main/java/com/cloudant/common/RetriableTask.java

```diff
@@ -70,6 +70,10 @@ public T call() throws Exception {
                     logger.fine("Success after retry: " + triesRemaining + ", " + task.toString());
```

Comment:
I realise this is unrelated to your current change so maybe should be another bug, but it is the first time I've seen this code:
`triesRemaining < 3` is wrong it should be `triesRemaining < totalRetries`
Also the logging messages are wrong, they should be
`logger.fine("Retry #: " + (totalRetries - triesRemaining) + "," + task.toString());`
and
`logger.fine("Success after retry: " + (totalRetries - triesRemaining) + ", " + task.toString());`
or maybe be reworded to be about tries remaining to potentially avoid the confusion of "Retry #0" being the first try.